pkg/routeagent_driver/main.go

```diff
@@ -73,6 +77,28 @@ func main() {
 		klog.Fatalf("Error building submariner clientset: %s", err.Error())
 	}
 
+	var localCluster *submarinerv1.Cluster
+
+	isGlobalnet := false
+	// During installation, sometimes creation of clusterCRD by submariner-gateway-pod would take few secs.
+	for i := 0; i < 100; i++ {
```

Comment:
I'd recommend the use of a retry lib instead of a loop:
[example](https://github.com/submariner-io/submariner/blob/devel/pkg/endpoint/public_ip.go#L124)

Reply:
OK, 10x.
Done